combined-notebook/OPTIMIZED__ALL_STT_QA_TTS_CURRENT_REFACTORED.ipynb

```diff
@@ -0,0 +1,1453 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "jd53z9Iphmaz"
+      },
+      "source": [
+        "# VARIABLES"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "ZAA4uc_Ghqlh"
+      },
+      "outputs": [],
+      "source": [
+        "from google.colab import userdata\n",
+        "\n",
+        "# populate the values in the secrets section\n",
```

Comment:
we will need to update the README to talk about the items here as well. @kojomensahonums can you please provide text to use for STT portions
@naafidasana please update README for QA related items as well here